tests/components/ista_ecotrend/test_sensor.py

```diff
@@ -14,6 +15,7 @@
 
 @pytest.mark.usefixtures("entity_registry_enabled_by_default")
 async def test_setup(
+    recorder_mock: Recorder,
```

Comment:
To avoid unused variable, I suggest that you move it to the decorator (and the same can be applied in test_init)
```suggestion
@pytest.mark.usefixtures("recorder_mock", "entity_registry_enabled_by_default")
async def test_setup(
```